packages/wonder-blocks-core/components/provider-group.js

```diff
@@ -0,0 +1,37 @@
+// @flow
+import * as React from "react";
+
+type Provider<P> = {|
+    type: React.ComponentType<P>,
+    props?: P,
+|};
+
+type Props = {|
+    providers: Array<Provider<any>>,
+    children: (Array<any>) => React.Node,
+|};
+
+export default class ProviderGroup extends React.Component<Props> {
```

Comment:
Rather than `ProviderGroup`, perhaps this should be called `CaaFConcatenator` or something. Thoughts?

Reply:
@kevinbarabash had suggested `ProviderComposer` in ADR-77. Might I suggest, `CaaFComposer`?

Reply:
`CaaFConcatenator` feels better.  `Provider` is a very overloaded term.